tests/variantstudy/model/command/test_update_general_config.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2025, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+from antarest.study.business.model.config.general_model import GeneralConfigUpdate
+from antarest.study.storage.rawstudy.model.filesystem.factory import FileStudy
+from antarest.study.storage.variantstudy.model.command.update_general_config import UpdateGeneralConfig
+from antarest.study.storage.variantstudy.model.command_context import CommandContext
+
+
+class TestUpdateGeneralConfig:
```

Comment:
We should also add some test in test_command_factory to check the serialization and parsing to DB.